core/src/main/scala/dagr/core/execsystem/TaskManager.scala

```diff
@@ -553,23 +553,26 @@ class TaskManager(taskManagerResources: SystemResources = TaskManagerDefaults.de
       allDone = graphNodesInStatesFor(List(ORPHAN, COMPLETED)).size == graphNodes.size
 
       if (!allDone && runningTasks.isEmpty && tasksToSchedule.isEmpty) {
-        logger.error(s"There are ${readyTasks.size} tasks ready to be scheduled but not enough system resources available.")
-        readyTasks.foreach { readyTask =>
-          val resourcesType: String = readyTask match {
-            case _: FixedResources    => "FixedResources"
-            case _: VariableResources => "VariableResources"
-            case _: Schedulable       => "Schedulable"
-            case _                    => "Unknown Type"
+        if (readyTasks.nonEmpty) {
```

Comment:
Ok, well, I'm going to leave this as-is then, because the whole point of this PR is that a message was getting logged that was causing confusion by indicating that the pipeline had failed due to one reason when it was another.  Orphans are tested for above.  I think the only reason to be inside the outer `if`, and not the inner `if` here is if there are one or more `FAILED` tasks, and that isn't a scheduling error, and is reported further below anyway.  If we find some case that's not being messaged appropriately lets add it then.